docs/send-data/kubernetes/best-practices.md

```diff
@@ -348,6 +364,9 @@ The `sumologic.com/sourceCategoryReplaceDash` annotation with value `-` prevents
 ### Excluding data using annotations
 
 You can use the `sumologic.com/exclude` [annotation](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) to exclude data from Sumo. This data is sent to the metadata enrichment service, but not to Sumo.
+You can exclude data per pod using pod annotations or per namespace using namespace annotations but pod annotations take precedence over namespace annotations.
+
+For example using pod annotations:
```

Comment:
so if I exclude namespace, I can still get data for pod using annotations?

Reply:
yes, tested with this configuration

```
apiVersion: v1
kind: Namespace
metadata:
  annotations:
    sumologic.com/exclude: "true"
  name: test-exclude
---
apiVersion: v1
kind: Pod
metadata:
  name: pod-a
  namespace: test-exclude
spec:
  containers:
    - name: container1
      image: public.ecr.aws/docker/library/busybox:latest
      args:
        - /bin/sh
        - -c
        - |
          while true; do
            echo "2022-02-08T20:53:28.621+0000 1st single line... pod-a"
            echo "2022-02-08T20:53:29.622+0000 2nd single line... pod-a"
            echo "2022-02-08T20:53:29.622+0000 3nd single line... pod-a"
            sleep 5
          done
---
apiVersion: v1
kind: Pod
metadata:
  name: pod-b
  namespace: test-exclude
  annotations:
    sumologic.com/include: "true"
spec:
  containers:
    - name: container1
      image: public.ecr.aws/docker/library/busybox:latest
      args:
        - /bin/sh
        - -c
        - |
          while true; do
            echo "2022-02-08T20:53:28.621+0000 1st single line... pod-b"
            echo "2022-02-08T20:53:29.622+0000 2nd single line... pod-b"
            echo "2022-02-08T20:53:29.622+0000 3nd single line... pod-b"
            sleep 5
          done

```

Reply:
I would mention this example explicitly 😅